src/Microsoft.ML.Data/Transforms/TransformsCatalog.cs

```diff
@@ -12,10 +12,29 @@ public sealed class TransformsCatalog
     {
         internal IHostEnvironment Environment { get; }
 
+        /// <summary>
+        /// The list of operations over categorical data.
+        /// </summary>
         public CategoricalTransforms Categorical { get; }
+
+        /// <summary>
+        /// The list of operations for rescaling data.
```

Comment:
>rescaling data [](start = 39, length = 14)

`data conversion` maybe `casting data` or `data casting`? Hashing, TypeConversion is hard to call rescaling operation.

Reply:
Good catch. This is actually an incorrect description left-over from when normalizers were here. I copy-pasted it, from the catalog, assuming it was correct. Will update it.

---
In reply to: [253240931](https://github.com/dotnet/machinelearning/pull/2382#discussion_r253240931) [](ancestors = 253240931)